packages/teams-js/src/public/geoLocation.ts

```diff
@@ -0,0 +1,132 @@
+import { sendAndHandleSdkError as sendAndHandleError } from '../internal/communication';
+import { ensureInitialized } from '../internal/internalAPIs';
+import { errorNotSupportedOnPlatform, FrameContexts } from './constants';
+import { DevicePermission, ErrorCode } from './interfaces';
+import { runtime } from './runtime';
+
+/**
+ * Namespace to interact with the geoLocation module-specific part of the SDK. This is the newer version of location module.
+ */
+export namespace geoLocation {
+  /**
+   * Data struture to represent the location information
+   */
+  export interface Location {
+    /**
+      Latitude of the location
+      */
+    latitude: number;
+    /**
+      Longitude of the location
+      */
+    longitude: number;
+    /**
+      Accuracy of the coordinates captured
+      */
+    accuracy?: number;
+    /**
+      Time stamp when the location was captured
+      */
+    timestamp?: number;
+  }
+  /**
+   * Fetches current user coordinates
+   * @returns Promise that will resolve with the {@link geoLocation.Location} object
+   */
```

Comment:
Should we include in our doc comments that the function could reject with an error, and also throw a not supported on platform error?

Reply:
This applies to all the functions in this file (and probably a bunch more files/capabilities that are definitely outside the scope of this PR).

Reply:
I have made the changes. Could you review again please?